packages/web/utils/function.ts

```diff
@@ -1 +1,10 @@
+import { isFunction } from "./assertion";
+
 export const noop = () => {};
+
+export function runIfFn<T, U>(
```

Comment:
This is fine, but I sense it's a sign of some over engineering happening somewhere.

Reply:
Indeed, I only intend to use this on custom headless components as it helps send state values through render props.